caikit/runtime/http_server/http_server.py

```diff
@@ -393,9 +409,9 @@ async def _handler(context: Request) -> Response:
                 error_content = {
                     "details": f"Unhandled exception: {str(err)}",
                     "code": error_code,
-                    "id": None,
+                    "id": uuid.uuid4().hex,
                 }
-                log.error("<RUN51881106E>", err, exc_info=True)
+                log.error("<RUN51231106E>", error_content, exc_info=True)
```

Comment:
@prashantgupta24 did we want a `log.error` in the `except CaikitRuntimeException` case as well?

Reply:
done

Reply:
hmm, duplicate logging can for sure be annoying...

if a quick test shows that the CREs are already logged with the id in a case or two, then it's probably fine to omit? We can always add it here later if we find the traceability hole

Reply:
Hmm, we log it most of the times, but not all 😞 I'm inclining towards having duplicate logs vs not having logs at all in some cases

Reply:
sounds good to me too!